code/style.css

```diff
@@ -2,6 +2,327 @@
+    
+  }
+
+  .input-box:active {
+    background-color: #FFE9D6;
+    border: black;
+  }
+
+  /* .input-box-label:valid {
+    background-color: #FFE9D6;
+    border: 5px solid #A7D0CD;
+  }
+
+  .input-box-label:invalid {
+    border: red;
+  } */
+
+
+
+  input[type="radio"] {
+  height: 2vh;
+  width: 2vh;
+  background-color: #FFE9D6;
+  
+}
+
+input[type="checkbox"]{
+  height: 2vh;
+  width: 2vh;
+  background-color: #FFE9D6;
+
+}
+
+button {
+    padding: 2vh;
+    border-radius: 10pt;
+    background-color: #7B6079;
+    color: #FFE9D6;;
+    font-size: 2vh;
+    width: 100%;
+    margin-top: 4vh;
+  }
+
+/* Img */
+img {
+   padding-top: 5vh;
+    width: 100%;
+    object-fit: contain;
+    height: auto;
+  }
+
+ 
+/* FAQ */
+
+.faq-content {
+padding-top: 10vh;
+min-height: 200px;
+ }
+.question {                         /* question FAQ */
+      background-color: #7B6079;
+      cursor: pointer;
+      width: 100%;
+      transition: 0.5s;
+      height: 15vh;
+      margin:10px 0 0 0;
+      box-sizing: border-box;
+      padding: 2vh;
+      text-align: center;
+      font-size: 3vh;
+      color:#FFE9D6;
+         
+  }
+
+  /* before clicking */ 
+.answer{                                   
+  display: none;
+}
+
+/* after clicking */
+.active + .answer {                         
+  box-sizing: border-box;
+  display: block;
+  background-color: #FFE9D6;
+  margin:0;
+  padding: 1vh 2vh;
+  height: 20vh;
+
+}
+
+.active{
+  background-color: grey;
+}
+/* Footer */
+footer{
+  box-sizing: border-box;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  height: auto;
+  padding: 4vh;
+  color: #FFE9D6;
 }
+
+.footer-icon {
+  width: 15vh;
+}
+  
+  /* --------------------TABLET---------------------------------*/
+
+@media (min-width: 720px){
+
+  .hero-video-phone {
+    display: none;
+  }
+  
+  .hero-video-tablet {
+    position: relative; 
+    right: 0;
+    width: 100%; 
+    height: 75vh;
+    display: block;
+    object-fit: cover;
+    
+  }
+  .hero-video-desktop {
+    display: none;
+  }
+
+
+  .main-content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto auto;
+    grid-gap: 20px;
+    padding: 20px;
+    
+  }
+
+  .sign-up-content{
+    grid-row: 1 / 3;
+    grid-column: 1;
+    padding: 5vh;
+  }
+
+  .faq-content {
+   grid-row: 2 / 4;
+   grid-column: 2;
+  }
+
+  .img-sign-up {
+    grid-row: 3;
+  }
+
+  .img-faq {
+    grid-row: 1;
+    grid-column: 2;
+  }
+
+  }
+  
+
+  /* --------------------Desktop---------------------------------*/
+
+@media (min-width: 1280px){
+ 
+  .hero-video-phone {
+    display: none;
+  }
+  
+  .hero-video-tablet {
+    display: none;
+  }
+  .hero-video-desktop {
+    position: relative; 
+    right: 0;
+    width: 100%; 
+    height: 75vh; 
+    display: block;
+    object-fit: cover;
+  }
+
+  .main-content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
```

Comment:
Grid-columns + rows at the same place. Looks like columns will just be overwritten by the rows?

Reply:
No, it is actually a grid of 2 *3 in desktop and tablet. But the forms are taking up to rows each